fish_prompt.fish

```diff
@@ -393,16 +393,15 @@ end
 
 function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   [ "$theme_display_ruby" = 'no' ]; and return
-  set -l ruby_version
-  if type rvm-prompt >/dev/null 2>&1
+  if which rvm-prompt >/dev/null 2>&1
     set ruby_version (rvm-prompt i v g)
-  else if type rbenv >/dev/null 2>&1
-    set ruby_version (rbenv version-name)
+  else if which rbenv >/dev/null 2>&1
+    set ruby_version (cat .ruby-version 2>/dev/null)
```

Comment:
Unfortunately, using just `.ruby-version` in the current directory won't give an accurate result. To properly emulate `rbenv version-name`, we would have to walk backwards up the current path and look for `.ruby-version` at every level. Do you have a sense for how much of the slowdown was related to this particular line? Is it worth re-implementing that logic in fish? Or would the speed improvements mentioned in the rbenv issue take care of this as of the next release anyway?